symengine/add.cpp

```diff
@@ -161,14 +157,13 @@ RCP<const Basic> Add::from_dict(const RCP<const Number> &coef, umap_basic_num &&
                 {
 #endif
                     // We need to copy the dictionary:
-                    map_basic_basic d2 =
-                        rcp_static_cast<const Mul>(p->first)->dict_;
+                    map_basic_basic d2 = rcp_static_cast<const Mul>(p->first)->dict_;
                     return Mul::from_dict(p->second, std::move(d2));
                 }
             }
             if (is_a<Pow>(*p->first)) {
                 insert(m, rcp_static_cast<const Pow>(p->first)->get_base(),
-                    rcp_static_cast<const Pow>(p->first)->get_exp());
+                       rcp_static_cast<const Pow>(p->first)->get_exp());
```

Comment:
whenever braking a function call, clang-format will align the arguments properly, i.e. the arguments in the new line will match the start of the arguments in the above line.

```
GOOD
my_func(mqwe, pter, lolqw,
               acc, cccs)

BAD
my_func(mssad, pwqe, loadsl,
           adsa, casd)
```